app/poros/food.rb

```diff
@@ -0,0 +1,18 @@
+class Food
+  attr_reader :id, :description, :brand, :ingredients
+
+  def initialize(data)
+    @id = data[:gtinUpc]
+    @description = data[:description]
+    @brand = data[:brandOwner]
+    # require "pry"; binding.pry
+    assign_ingredients(data[:ingredients])
+  end
+  def assign_ingredients(ingredients)
+    if ingredients == ""
+      @ingredients = "N/A"
+    else
+      @ingredients = ingredients
+    end
+  end
+end
```

Comment:
Instead of calling to a method in your initizlize method, you could do something like this: 

```ruby
class Food
  attr_reader :id, :description, :brand

  def initialize(data)
    @id = data[:gtinUpc]
    @description = data[:description]
    @brand = data[:brandOwner]
    @ingredient_data = data[:ingredients]
  end
  def ingredients
    if @ingredient_data == ""
      return "N/A"
    else
      return @ingredient_data
    end
  end
end
```

Reply:
Alternatively, I'd consider the logic in the `assign_ingredients` method to be presentation logic, and let my view be responsible for it.